app/src/main/res/layout/fragment_third_party_transfer.xml

```diff
@@ -241,6 +241,8 @@
                         android:layout_marginTop="10dp"
                         android:layout_weight="0.5"
                         app:backgroundTint="@color/white"
+                        android:textColor="@color/primary"
+                        style="@style/Widget.AppCompat.Button.Borderless"
```

Comment:
@ivary43 Remove this line, we want a border.

Reply:
Okay i will do it .one thing more the cancel button is not working .